RFXCom binding throws ClassNotFoundException for DatatypeConverter

The following exception is thrown when the binding accessed the USB port to receive data:

 Exception in thread "Thread-39" java.lang.NoClassDefFoundError: javax/xml/bind/DatatypeConverter at org.openhab.binding.rfxcom.internal.connector.RFXComSerialConnector$SerialReader.run(RFXComSerialConnector.java:178)
Caused by: java.lang.ClassNotFoundException: javax.xml.bind.DatatypeConverter cannot be found by org.openhab.binding.rfxcom_2.0.0.b1
at org.eclipse.osgi.internal.loader.BundleLoader.findClassInternal(BundleLoader.java:432)
at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:345)
at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:337)
at org.eclipse.osgi.internal.loader.ModuleClassLoader.loadClass(ModuleClassLoader.java:160)
at java.lang.ClassLoader.loadClass(ClassLoader.java:358)

See also the discussion at
https://community.openhab.org/t/rfxcom-binding-throws-java-lang-noclassdeffounderror-javax-xml-bind-datatypeconverter/6088

--- a/addons/binding/org.openhab.binding.rfxcom/META-INF/MANIFEST.MF
+++ b/addons/binding/org.openhab.binding.rfxcom/META-INF/MANIFEST.MF
@@ -9,6 +9,7 @@ Bundle-ClassPath: .,
  lib/jd2xx.jar
 Import-Package: com.google.common.collect,
  gnu.io,
+ javax.xml.bind,
  org.apache.commons.io,
  org.eclipse.smarthome.config.core,
  org.eclipse.smarthome.config.discovery,

